api/client.go

```diff
@@ -176,26 +176,40 @@ func (c *Client) Scheme() string {
 	return c.url.Scheme
 }
 
-// WaitForAPI polls the /v1/status endpoint to check when the CTS API is
-// available. The API is started when CTS is run as a daemon and completes
-// all tasks once.
-func (c *Client) WaitForAPI(timeout time.Duration) error {
+// WaitForOnce checks if all the current enabled tasks have been run once by
+// polling the /v1/status/tasks endpoint. Tasks that have not run will have an
+// unknown status. Should only use if no runtime changes are being actively made.
+func (c *Client) WaitForOnce(timeout time.Duration) error {
```

Comment:
Rather than `WaitForOnce` or `WaitForAPI` maybe we can make the name more about what we're waiting to be able to do, rather than waiting for? That way in the future if this were to change again, we wouldn't have to rename the function.

I can't think of a good name at the moment, but something that conveys "waiting to be able to process tasks" or "waiting to start the nominal workflow"

Reply:
Notes from chatting offline: went with "WaitForTestReadiness"

Other brainstormed ideas:

- WaitForReadiness
- WaitForTestReadiness
- WaitForTesting
- WaitToTest